llama_stack/apis/inference/inference.py

```diff
@@ -1051,7 +1051,7 @@ async def completion(
         """
         ...
 
-    @webmethod(route="/inference/batch-completion", method="POST", experimental=True, level=LLAMA_STACK_API_V1)
+    @webmethod(route="/inference/batch-completion", method="POST", level=LLAMA_STACK_API_V1)
```

Comment:
skip this, it'll cause a conflict in https://github.com/llamastack/llama-stack/pull/3261